docs/crio.conf.5.md

```diff
@@ -129,7 +129,7 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
  List of default sysctls. If it is empty or commented out, only the sysctls defined in the container json file by the user/kube will be added.
 
 **additional_devices**=[]
-  List of additional devices. If it is empty or commented out, only the devices defined in the container json file by the user/kube will be added.
+  List of additional devices. Specified as "<device-on-host>:<device-on-container>:<permissions>", for example: "--device=/dev/sdc:/dev/xvdc:rwm". If it is empty or commented out, only the devices defined in the container json file by the user/kube will be added.
```

Comment:
Stretching my volume knowledge a bit here.  is 'm' in ':rwm' valid?  I don't recall it off the top of my head.

Reply:
Oops, I also have to change this to additional-devices, not --device. I just copied it from the original PR, but I do think m is valid, I've seen that before.